internal/ingress/annotations/authtls/main.go

```diff
@@ -45,6 +52,9 @@ type Config struct {
 	ValidationDepth    int    `json:"validationDepth"`
 	ErrorPage          string `json:"errorPage"`
 	PassCertToUpstream bool   `json:"passCertToUpstream"`
+	OCSP               string `json:"ocsp"`
```

Comment:
Why assigning a json tag for it?

Reply:
Consistency. This is my first contribution and most of the other config objects I saw in had json tags on it. So I did it here too. 

https://github.com/kubernetes/ingress-nginx/blob/c17506b9fb071a4fbd27231ee7ef347f191206d3/internal/ingress/annotations/auth/main.go#L49-L57